### PR TITLE
Add commands for create table, add columns and filter info

### PR DESCRIPTION
**Pull request by Gdiazdiaz**
## Tasks completed

- Set table 'animals'
- Set columns for id, name, DOB, weight, neutered, escapes
- Set queries to filter data according to the following instructions

*Find all animals whose name ends in "mon".*
name than ends with 'mon'

*List the name of all animals born between 2016 and 2019.*
born btween

*List the name of all animals that are neutered and have less than 3 escape attempts.*
escapes less than 3 and neutered

*List the date of birth of all animals named either "Agumon" or "Pikachu".*
pikachu and agumon

*List name and escape attempts of animals that weigh more than 10.5kg*
more than 10kg

*Find all animals that are neutered.*
neutered

*Find all animals not named Gabumon.*
not gabumon

*Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)*
weight

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES (1, 'Agumon', '2020/02/03', 0, false, 10.23);
+INSERT INTO animals VALUES (2, 'Gabumon', '2018/11/15', 2, true, 8);
+INSERT INTO animals VALUES (3, 'Pikachu', '2021/01/07', 1, false, 15.04);
+INSERT INTO animals VALUES (4, 'Devimon', '2017/05/12', 5, true, 11);
```
Comment (mikethreels) [inline data.sql]: - [ ] when inserting into the table you need to specify which column as well like:
```
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23');
```

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Glen, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Gdiazdiaz) [reply data.sql]: Hi @mikethreels, I appreciate the feedback, however I don't need to specify wich column as long as I write the values in order, Is there any consideration I need to have in mind that make specify the column imperative?
Thank you for taking the time to answer my question.

**Review by uwadonat — APPROVED** [review]
## Hi @Gdiazdiaz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- You are using git-flow correctly ✔️
- Well organized project ✔️
209362895-9762f2f8-13cf-4ce0-90b6-a672c4f04196

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**